packages/components/molecules/f-card-with-content/src/components/CardWithContent.vue

```diff
@@ -1,32 +1,107 @@
 <template>
-    <div
-        :class="$style['c-cardWithContent']"
-        data-test-id="cardWithContent">
-        I am a CardWithContent Component (GB)
-    </div>
+    <card-component
+        has-outline
+        is-page-content-wrapper
+        data-test-id="cardWithContent"
+        :class="$style['c-cardWithContent']">
+        <div
+            v-if="$slots.icon"
+            :class="$style['c-cardWithContent-icon']">
+            <slot name="icon" />
+        </div>
+
+        <h1
+            v-if="cardHeading"
+            :class="$style['c-cardWithContent-heading']">
+            {{ cardHeading }}
```

Comment:
What if you need to add markup to a heading, for example, I've needed to add `span` tags to heading a fair bit recently to allow for a mix of styles or hidden a11y content. It's not really possible with `prop`s, but you can do it with `slot`s. This is probably more evident with the cardDescription` below — you're limited to passing a single string of text that will be rendered into a single paragraph.

Pretty sure I've had this conversation elsewhere at some point but I'd always use slots to add content as they are much more flexible. In fact, I see you're using one above for the icon.

Reply:
Yeah I see the need for that, and this can always be changed later as the use cases change, this definitely isn't set in stone, it does just enough for our use case at the moment.

The difficulty has been how far do we go with swapping everything out with slots? If we replace everything with a slot then we're effectively back at f-card, and we'd need to duplicate the common styling wherever this type of card is used.